src/renderer/components/modals/create-note-modal/CreateNoteModal.html

```diff
@@ -24,6 +24,15 @@ <h2 class="text-red" v-if="displayDupError">Oh sorry, you can't have duplicated
         </b-input>
       </b-field>
 
+      <b-field horizontal label="Source">
```

Comment:
Maybe this `v-model` could be renamed to `note.sourceUrl`, what do you think?

Reply:
Done!